### DISCUSSION TITLE
Folder/Directory Size?

### DISCUSSION THREAD

**Discussion by assassins21** [sixem/ivfi-php#21] [Ideas]
Is it possible to show the folder size as well like the file size?

**Comment by sixem** [comment]
It's possible in theory, of course, but it would require the script to scan all the files of each directory visible, then add all the file sizes of each file in the respective directory together in order to get the directory size. It would just be very clunky and I'm afraid it can't be done in a good and efficient way.

It would've been nice to have, though, had there only been an easier way of achieving it. :/

**Comment by sixem** [comment]
Actually, if I have some time later, I can try to implement this and see how the performance is. If it isn't too bad, then maybe I can make it an option.

**Reply by assassins21** [reply]
Well thank you for considering it :D 
I hope it can be managed!

**Reply by sixem** [reply]
I've added it as an option. You can check it on the [dev-1.1.8](https://github.com/sixem/eyy-indexer/tree/dev-1.1.8) branch.

I have to test it out a bit more and work on some other changes before I publish it as an official release though.

**Reply by assassins21** [reply]
That's great, it works just fine
I will be waiting for the official release
